dashboard/src/pages/packages/index.vue

```diff
@@ -52,6 +66,26 @@ const toggleLegend = () => {
       {{ packageStore.page.total }}
     </div>
 
+    <div class="row mx-0 mb-3 align-items-left border bg-light">
+      <div class="col-auto pb-1 align-self-center border-end">
+        <h3 class="mb-0">Filters</h3>
+      </div>
+      <div class="col-auto pb-1">
+        <label for="filter-status" class="form-label mb-0"> Status </label>
+        <select
+          id="filter-status"
+          v-model="statusFilter"
+          @change="changeStatusFilter"
+          class="form-select border rounded-2"
```

Comment:
`class="form-select"` should be enough considering my previous comment about the boxy look.